### PR TITLE
b-402: Clarifying what Job HP and MP bonuses are

### PR DESCRIPTION
**Pull request by BrunoCarvalhoPaula**
Clarifying what Job HP and MP bonuses are.

### CODE REVIEW

File: core/revised/gmguide.tex [hunk 1]
```diff
@@ -207,21 +207,22 @@ \subsubsection{Environment Effects}\label{gm-enveffct}
...
-%
-%    \adjincludegraphics[height=0.4\textheight]{block-moogle}
-%
-%    \vspace{\stretch{1}}
-%
-%    \textcolor{mogred}{\arabtype\LARGE This page intentionally left blank, kupo!}
-%
-%    \vspace*{\stretch{1}}
-%\end{center}
+\begin{center}
```
Comment (atmafox) [inline core/revised/gmguide.tex]: Should include the interntional-blank here like it was in core/revised.tex

**Review by atmafox — CHANGES_REQUESTED** [review]
Please see comments for requested change!

File: core/revised/gmguide.tex (hunk 1 above)
Comment (atmafox) [reply core/revised/gmguide.tex]: I made these changes

**Review by WeaverThree — DISMISSED** [review]
Seems good

**Review by WeaverThree — APPROVED** [review]

**Review by atmafox — APPROVED** [review]
LGTM now!

**Pull request merged**